apps/andi/lib/andi_web/controllers/edit_controller.ex

```diff
@@ -0,0 +1,17 @@
+defmodule AndiWeb.EditController do
+  use AndiWeb, :controller
+  alias Andi.DatasetCache
+
+  def show(conn, %{"id" => id}) do
+    case DatasetCache.get(id) do
+      %{"dataset" => dataset} ->
+        live_render(conn, AndiWeb.EditLiveView, session: %{dataset: dataset})
+
+      _ ->
```

Comment:
404s are logged by phoenix by default. It feels to me like we shouldn't be adding additional logging when users try to hit a non existing route. Though I can see the point about this occuring because of an error with our event handling or caching being worth logging. However, I wouldn't be able to differentiate between the two causes at this point in the code.

TLDR: I wouldn't log this but I could be convinced otherwise.